resources/evaluate.q

```diff
@@ -173,7 +173,7 @@
   (generateColumns[::;0b;1b;key data;"key"]; generateColumns[::;0b;0b;value data;"values"]);
   isTable;
   generateTableColumns[originalType;isAtom;0b;data];
-  generateColumns[originalType;isAtom;0b;data;$[isAtom;"value";"values"]]
+  generateColumns[originalType;isAtom;0b;data;"values"]
```

Comment:
Line 176 not always returns an array as stated in the ticket, `enlist` is missing.